--- Makefile.coq.orig	2021-10-04 11:15:22.592822933 +0200
+++ Makefile.coq	2021-10-04 11:17:18.684584261 +0200
@@ -480,6 +485,9 @@
 	$(HIDE)mkdir -p html
 	$(HIDE)$(COQDOC) \
 		-toc $(COQDOCFLAGS) -html $(GAL) $(COQDOCLIBS) -d html $(VFILES)
+# Prosa hack: let's tweak the style a bit
+	$(HIDE)sed -i.bak "s/#90bdff/#eeeeee/" html/coqdoc.css
+	$(HIDE)rm html/coqdoc.css.bak
 
 mlihtml: $(MLIFILES:.mli=.cmi)
 	$(SHOW)'CAMLDOC -d $@'
